frontend/src/app/features/assistance-selection/assistance-selection.component.spec.ts

```diff
@@ -0,0 +1,41 @@
+import { ComponentFixture, TestBed } from '@angular/core/testing';
+import { AssistanceLevel } from 'src/app/core/interfaces/assistance-level';
+import { AssistanceSelectionComponent } from './assistance-selection.component';
+
+describe('AssistanceSelectionComponent', () => {
+  let component: AssistanceSelectionComponent;
+  let fixture: ComponentFixture<AssistanceSelectionComponent>;
+
+  beforeEach(async () => {
+    await TestBed.configureTestingModule({
+      declarations: [ AssistanceSelectionComponent ]
+    })
+    .compileComponents();
+
+    fixture = TestBed.createComponent(AssistanceSelectionComponent);
+    component = fixture.componentInstance;
+    fixture.detectChanges();
+  });
+
+  it('should create', () => {
+    expect(component).toBeTruthy();
+  });
+
+  it('handleSelectLevel should change the selected level', () => {
+    const comp = new AssistanceSelectionComponent();
+    expect(comp.selectedLevel).withContext('will have nothing selected at first').toBe('')
+
```

Comment:
Here, you've got an expect in the middle of the test. If you're doing this, make it two tests and the `withContext` isn't needed. Also, I generally prefer `toEqual` rather than `toBe` ... it's more explicit and more consistent across test frameworks.